apps/explorer/src/app/components/order-details/amend-order-details.spec.tsx

```diff
@@ -223,7 +225,8 @@ describe('Amend order details', () => {
     expect(await res.findByText('-7879')).toBeInTheDocument();
   });
 
-  it('Fetches latest version when version is not specified', async () => {
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip('Fetches latest version when version is not specified', async () => {
```

Comment:
@edd couldn't figure out why this wasn't working, if you have a mo can you take a look?

Reply:
Looking...

Reply:
Fixed and re-enabled. Unsure why this change raised the problem, but mocking an extra GraphQL call fixed the test.